### ISSUE TITLE
PopupMenuButton lacks sufficient text contrast

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#19961]
Ratio is currently 3.36, should be at least 4.50.

- Foreground: `#8C8C8C`
- Background: `#FFFFFF`

**Comment by raacker** [comment]
@jonahwilliams  I'd like to try this! WIP

**Comment by jonahwilliams** [comment]
Hi @raacker I'm sorry I completely missed your comment - this is for our Flutter Gallery specifically, so its more a matter of design choice. If you would like to contribute to any of the accessibility bugs we have though I would be happy to point you towards some good first issues.

**Comment by raacker** [comment]
@jonahwilliams  It's okay. Thanks for reply. I tried to look into this issue and I figured out ratio was fine! :) And sure! it would be my pleasure to start any issues  👍
